### PR TITLE
STU-268 Export Test Result to Cloud | Visual in Column

### PR DESCRIPTION
**Pull request by LatentDream**
# Demo
https://github.com/flojoy-ai/studio/assets/56351875/151669bb-a058-4ac0-8ce3-486215746db8

`test_that_pass_1` is not in registered in the cloud.
Better to have:
- An error message ? 
- Creating the test automatically ?

### CODE REVIEW

File: src/renderer/routes/test_sequencer_panel/components/DataTable.tsx [hunk 1]
```diff
@@ -205,16 +209,21 @@ export function DataTable() {
...
-    //     ) : null;
-    //   },
-    // },
+    {
+      accessorFn: (elem) => {
+        return elem.type === "test" ? "isSavedToCloud" : null;
+      },
+      header: "Saved To Cloud",
+      cell: ({ row }) => {
+        const cloudStatus =
```
Comment (smahmed776) [inline src/renderer/routes/test_sequencer_panel/components/DataTable.tsx:218]: Can be simplified to:

```suggestion
               return row.original.type === "test" ? (
          <div
            className={cn({
              "text-green-500": row.original.isSavedToCloud,
              "text-red-500": !row.original.isSavedToCloud,
            })}
          >
            {row.original.isSavedToCloud ? "Saved" : "Not Saved"}
          </div>
        ) : null;
```

**Review by smahmed776 — CHANGES_REQUESTED** [review]
Just cosmetic thing, otherwise looks good...

File: captain/utils/test_sequencer/run_test_sequence.py [hunk 2]
```diff
@@ -240,18 +245,30 @@ def reverse_id(test_name) -> str:
             )
             node.is_saved_to_cloud = True
             logger.info(f"{test_name}: Uploaded to cloud")
-        except KeyError as err:
-            # TODO: Return error to user
+        except KeyError:
```
Comment (smahmed776) [inline captain/utils/test_sequencer/run_test_sequence.py:248]: Just to verify, does `client.upload` throw `KeyError` if a test was not found in Cloud under given project?

File: src/renderer/routes/test_sequencer_panel/components/DataTable.tsx [hunk 3]
```diff
@@ -82,6 +81,11 @@ const mapStatusToDisplay: { [k in StatusTypes]: React.ReactNode } = {
   pending: <p className="text-yellow-500">PENDING</p>,
 };
 
+const mapCloudStatusToDisplay: { [k in "true" | "false"]: React.ReactNode } = {
```
Comment (smahmed776) [inline src/renderer/routes/test_sequencer_panel/components/DataTable.tsx:84]: Redundant map object.

**Review by smahmed776 — CHANGES_REQUESTED** [review]

File: captain/utils/test_sequencer/run_test_sequence.py (hunk 2 above)
Comment (LatentDream) [reply captain/utils/test_sequencer/run_test_sequence.py:248]: Yes!

**Review by izi-on — APPROVED** [review]
LGTM

**Review by smahmed776 — APPROVED** [review]
Looks good

**Pull request merged**